rfc-126-custom-configuration-for-dependabot.md

```diff
@@ -233,15 +235,15 @@ This would allow us to remove the hardest PRs from updates, but it would require
 
 [dependabot-ignored-updates]: https://dependabot.com/docs/config-file/#ignored_updates
 
-## Consequences and measuring success
+## Consequences, risks and measuring success
 
 Using [the statistics from above](#Statistics), we can make some estimates on what affect this would have had if it had been enabled from the beginning:
 
 - 7378 PRs wouldn't have been raised (10216 rather than the 17526 we've had so far).
 - The mean time to merge would have decreased from almost 3 days to just under 2 days.
 - The ignored libraries alone would have taken on average 3.5 days to merge.
 
-In theory, by limiting the number of libraries that get updated by Dependabot, our applications would end up less up to date. However, by prioritising certain libraries over others, we should end up in a situation where our applications are actually _more_ up to date in the areas we care about (security, frameworks, internal libraries).
+In theory, by limiting the number of libraries that get updated by Dependabot, our applications would end up less up to date. However, by prioritising certain libraries over others, we should end up in a situation where our applications are actually _more_ up to date in the areas we care about (security, frameworks, internal libraries). There is a risk involved in not being able to access new features or fixes from ignored libraries, however the reason the library has been ignored is because we're making little use of it in the app (otherwise it should be a framework library). We can always manually upgrade these libraries when we want to, and that should still be encouraged when doing a major update (for example a Rails upgrade).
```

Comment:
I could have been even more helpful and said: another risk is that people find the global config confusing, since it will list gems that aren't relevant to the repo. The impact is that people may delete pieces of the config, only to have a PR raised back at them. This seems pretty easy to address, by including an explanatory comment.

Perhaps this is just an implementation detail, more than a risk!

Reply:
Yeah, I think this is worth mentioning. I've added a sentence in 19c6952555d3d39563ea49286a6df361d502e9b1.